concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
+RonaTas-2011-431-4	4	stream, current	2935	CURRENT (STREAM)
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
```

Comment:
If we enforce unique mappings, I'd probably unmap this `ROT` instead of the second one (ID 13).

Reply:
Resolved in d5237b728dd839b9352f77f30508546e248f53f4